misc/dist/osx_tools.app/Contents/Info.plist

```diff
@@ -90,6 +90,57 @@
 				<string>scene/x-scn</string>
 			</dict>
 		</dict>
+		<dict>
+			<key>UTTypeIdentifier</key>
+			<string>public.gd</string>
+			<key>UTTypeReferenceURL</key>
+			<string></string>
+			<key>UTTypeDescription</key>
+			<string>Godot Script</string>
+			<key>UTTypeIconFile</key>
+			<string>GDScript.icns</string>
+			<key>UTTypeConformsTo</key>
+			<array>
+				<string>public.data</string>
+			</array>
+			<key>UTTypeTagSpecification</key>
+			<dict>
+				<key>com.apple.ostype</key>
+				<string>GDSC</string>
+				<key>public.filename-extension</key>
+				<array>
+					<string>gd</string>
+				</array>
+				<key>public.mime-type</key>
+				<string>script/x-gd</string>
+			</dict>
+		</dict>
+		<dict>
+			<key>UTTypeIdentifier</key>
+			<string>public.res</string>
+			<key>UTTypeReferenceURL</key>
+			<string></string>
+			<key>UTTypeDescription</key>
+			<string>Godot Script</string>
+			<key>UTTypeIconFile</key>
+			<string>Resource.icns</string>
+			<key>UTTypeConformsTo</key>
+			<array>
+				<string>public.data</string>
+			</array>
+			<key>UTTypeTagSpecification</key>
+			<dict>
+				<key>com.apple.ostype</key>
+				<string>GDRC</string>
```

Comment:
Does the specification require such short identifiers? I know the existing one for the `project.godot` file was `GODP`, but it seems pretty cryptic to me. If we can use longer identifiers, something like `Godot Resource` would be better IMO (or `GodotResource` if spaces are an issue).

Reply:
OSType is 4-byte, but these ID are only used for Mac OS Classic types, and can be removed at all.